### PR TITLE
Add project guidelines and issues/prs templates

### PR DESCRIPTION
**Pull request by caiovfernandes**
This pull request push project guidelines and templates files to improve repository community health.

## Types of changes
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+* **Include screenshots and animated GIFs** which show you following the described steps and clearly demonstrate the problem. If you use the keyboard while following the steps, **record the GIF with the [Keybinding Resolver](https://github.com/atom/keybinding-resolver) shown**. You can use [this tool](https://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://gitlab.gnome.org/Archive/byzanz) on Linux.
+* **If you're reporting that the module crashed**, include a crash report with a stack trace from the operating system. Include the crash report in the issue in a [code block](https://docs.github.com/pt/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks), a [file attachment](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/), or put it in a [gist](https://gist.github.com/) and provide a link to that gist.
+* **If the problem is related to performance or memory**, include a [CPU profile capture](https://flight-manual.atom.io/hacking-atom/sections/debugging/#diagnose-runtime-performance) with your report.
```
Comment (arthurbdiniz) [inline CONTRIBUTING.md:28]: Not related

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+* **Can you reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens.
+* If the problem is related to working with files (e.g. opening and editing files), **does the problem happen for all files and projects or only some?** Does the problem happen only when working with local or remote files (e.g. on network drives), with files of a specific type (e.g. only JavaScript or Python files), with large files or files with very long lines, or with files in a specific encoding? Is there anything else special about the files you are using?
+
+Include details about your configuration and environment:
+
+* **Which version of Terraform are you using?** You can get the exact version by running `terraform -v` in your terminal.
+* **What's the name and version of the OS you're using**?
+* **Are you running Terraform in a virtual machine?** If so, which VM software are you using and which operating systems and versions are used for the host and the guest?
+* **Which keyboard layout are you using?** Are you using a US layout or some other layout?
```
Comment (arthurbdiniz) [inline CONTRIBUTING.md:43]: Can remove this one.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+## Pull Request Process
+
+The process described here has several goals:
+
+- Fix problems that are important to users.
+- Engage the community in working toward the best possible terraform module.
+- Enable a sustainable system for DNXers maintainers to review contributions.
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
```
Comment (arthurbdiniz) [inline CONTRIBUTING.md:73]: can be removed.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+- Engage the community in working toward the best possible terraform module.
+- Enable a sustainable system for DNXers maintainers to review contributions.
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+5. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
```
Comment (arthurbdiniz) [inline CONTRIBUTING.md:79]: Not linked correctly to the template.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+6. Follow the [styleguides](#styleguides)
```
Comment (arthurbdiniz) [inline CONTRIBUTING.md:80]: include Terraform styleguides.
You can take from this one https://github.com/jonbrouse/terraform-style-guide/blob/master/README.md
To not make this file to big, create a new file with the styleguides inside the repo and link it here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (caiovfernandes) [reply CONTRIBUTING.md]: Done ✅

File: CONTRIBUTING.md (hunk 2 above)
Comment (caiovfernandes) [reply CONTRIBUTING.md]: Done ✅

File: CONTRIBUTING.md (hunk 3 above)
Comment (caiovfernandes) [reply CONTRIBUTING.md:59]: Done ✅

File: CONTRIBUTING.md (hunk 4 above)
Comment (caiovfernandes) [reply CONTRIBUTING.md]: Done ✅

File: CONTRIBUTING.md (hunk 5 above)
Comment (caiovfernandes) [reply CONTRIBUTING.md]: Done ✅

**Review by arthurbdiniz — APPROVED** [review]

**Pull request merged**